src/components/sw360/Header/Header.tsx

```diff
@@ -80,7 +80,7 @@ const Header = () => {
 
             {session ? (
                 <div className='row'>
-                    <Navbar />
+                    <SW360Navbar />
```

Comment:
Navbar is not shown on login page so we don't need check the session. The useSession take a few second to get the session and it will make Navbar rendered later than the page's body.

Reply:
DONE